tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -663,6 +663,20 @@ public void TestCreateCollectionSetCappedSetMaxSize()
         [Fact]
         public void TestCreateIndex()
         {
+            void AssertNamespace(IndexInfo indexInfo)
```

Comment:
I didn't realize on first review that this was a local function.

Therefore AssertNamespace is actually a variable.

Should probably be lowercase.

Feel free to comment if you disagree. I won't hold up approval based on this comment alone.

Reply:
Uppercase naming was suggested by resharper rules (not sure which rules are used in our code). But probably you are right. Fixed.